packages/orchestration/src/utils/start-helper.js

```diff
@@ -55,7 +56,7 @@ export const provideOrchestration = (
     zone,
     makeRecorderKit,
     zcf,
-    remotePowers.timerService,
+    timerService,
     vowTools,
     chainHub,
```

Comment:
Aside: do we really have a function with that many positional args? If it gets >3, they should be named.